api/tests/unit/environments/test_unit_environments_views.py

```diff
@@ -949,19 +952,30 @@ def test_delete_trait_keys_deletes_traits_matching_provided_key_only(
     assert Trait.objects.filter(identity=identity, trait_key=trait_to_persist).exists()
 
 
+@pytest.mark.parametrize(
+    "client",
+    [(lazy_fixture("admin_master_api_key_client")), (lazy_fixture("admin_client"))],
+)
 def test_user_can_list_environment_permission(
-    admin_client_new: APIClient, environment: Environment
+    client: APIClient, environment: Environment
```

Comment:
I don't understand why `admin_client_new` was replaced with the fixture since it had those fixtures anyway.

Reply:
Oh, that's my bad